docker-images/Dockerfile.oni-runtime-debug

```diff
@@ -12,8 +12,10 @@ RUN go get github.com/filecoin-project/go-paramfetch/paramfetch
 COPY /proof-parameters.json /
 RUN paramfetch 2048 /proof-parameters.json
 
+ARG LOTUS_COMMIT=e21ea53
```

Comment:
I think we might be better served setting this value as an argument in the `Makefile`.